modules/extensions-spring-support/src/test/java/org/mule/test/module/extension/validation/InvalidExtensionConfigTestCase.java

```diff
@@ -142,6 +142,8 @@ public void petStoreExclusiveGroupInsidePojoOneRequired() throws Exception {
 
   @Test
   public void petStoreExclusiveParameterRequired() throws Exception {
+    // FIXME W-10831629: there is an inconsistency between the element referred to by this error message and the one where there
+    // are conflicting exclusive optionals
```

Comment:
The fix for [W-10831629](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00000sWyAZYA0/view) should also affect this case.